Doppler.Currency/Startup.cs

```diff
@@ -153,12 +148,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
                 endpoints.MapControllers();
             });
 
-            app.UseSwagger();
 
-            app.UseSwaggerUI(c =>
+            if (env.IsDevelopment())
```

Comment:
Why only in development?

Reply:
Swagger needs set up correctly when a reverse proxy is used, I think that is not necessary Swagger in Prod, int, or QA, what do you think?, if It is necessary for prod, int and QA I'll create a task for it

Reply:
Oh, I understand, maybe you can add a comment in the code indicating why it is disabled.

Reply:
Done